Offer DID in BigInt format (#16)
rarimo/rarime: CHANGELOG.md

### Changed
- `@rarimo/rarime`:
  - Save credentials to ceramic instead of snap store

>   - Return did in bigint representation within default one in createIdentity method

Reviewer: ```suggestion
  - Return did in bigint representation alongside the default representation```
